spec/controllers/api/v1/packages_controller_spec.rb

```diff
@@ -649,6 +649,30 @@ def test_packages_location_changes(package)
       end
     end
 
+    context 'package notes' do
+      let(:en_note) { 'A sample note' }
+      let(:zh_note) { '如此申請不再受惠於個案受惠者，你可刪除受惠者資料。' }
+
+      it 'creates package with notes in en and zh_tw languages' do
+        package_params[:notes] = en_note
+        package_params[:notes_zh_tw] = zh_note
+        post :create, params: { package: package_params }
+        expect(parsed_body['package']['notes']).to eq(en_note)
+        expect(parsed_body['package']['notes_zh_tw']).to eq(zh_note)
+      end
+
+      context 'if english note is empty' do
```

Comment:
we probably want to test the flip side of things as well

```ruby
      context 'if only the zh_tw note is empty' do
          it 'creates the package successfully' do
          end
      end
```

Not a big deal, but as a habit we want to always test the different permutations

Reply:
sure